corehq/apps/dropbox/models.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from django.db import models
+from django.contrib.auth.models import User
+
+from .exceptions import DropboxUploadAlreadyInProgress
+from .tasks import upload
+
+
+class DropboxUploadHelper(models.Model):
+    """
+    This class is for working with dropbox uploads. It saves metadata to the database, stores the progress of
+    the upload, and ensures multiple uploads of the same file does not occur.
+    """
+    dest = models.CharField(max_length=255)
+    src = models.CharField(max_length=255)
+    progress = models.DecimalField(default=0, decimal_places=2, max_digits=3)
+    download_id = models.CharField(max_length=255)
+    user = models.ForeignKey(User)
```

Comment:
This is slightly different from what we normally do. we usually link to the string username and then look up the user with that string. not sure if you have an opinion on what route to take